internal/common/common_linux.go

```diff
@@ -30,8 +30,13 @@ func DoSysctrl(mib string) ([]string, error) {
 	return values, nil
 }
 
+// Deprecated: use NumProcsWithContext instead
```

Comment:
Could you remove this `Deprecated` also?

Reply:
Done.